development/setup.js

```diff
@@ -17,7 +17,7 @@ const createUser = (username, password, admin) =>
       `--email ${username}@digdir.no`,
       admin ? `--admin` : undefined,
       `--must-change-password=false`,
-    ].join(' ')
```

Comment:
For some reason, this file is formatted on commit. I've tried to roll it back, but it is formatted back again on commit.